pybossa/view/projects.py

```diff
@@ -309,6 +310,16 @@ def project_index(page, lookup, category, fallback, use_count, order_by=None,
         template_args.update({"count": count})
     return handle_content_type(template_args)
 
+def set_product_subproduct_choices(form):
```

Comment:
Could this be done in the `__init__` if you pass in the lists of products?

Reply:
So keep it as it was before but passing the products?